pkgs/development/interpreters/wamr/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wamr";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "bytecodealliance";
+    repo = "wasm-micro-runtime";
+    rev = "WAMR-${version}";
+    hash = "sha256-RO5GBwiozdMaLt2m6rUlw+UohnYlba6llrVudAsIHW8=";
+  };
+  dontInstall = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  configurePhase = ''
+    runHook preConfigure
+    mkdir -p build
+    cd build
+    cmake ../product-mini/platforms/linux
+    runHook postConfigure
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    mkdir -p $out/bin
+    make
+    cp iwasm $out/bin
```

Comment:
This should be in installPhase.

Reply:
+1, line 31 should be installPhase too. 
Do you even need a buildPhase? If you just calling make. I belive mkDerivation will call make automatically if you dont specify buildPhase

Reply:
Yes, my bad, I had `dontInstall = true;` in my first attempt, and then installed the binary in the build phase, which was wrong. I now removed `dontInstall = true;`, and just install the binary during the `installPhase`. Sorry, my bad.